formula-android-compose/build.gradle

```diff
@@ -27,9 +27,10 @@ android {
     }
 
     composeOptions {
-        kotlinCompilerVersion "1.5.10"
-        kotlinCompilerExtensionVersion "1.0.0-beta08"
+        kotlinCompilerVersion constants.kotlinVersion
+        kotlinCompilerExtensionVersion composeVersion
```

Comment:
note: I tried to remove this, it was a big mistake .. I spent a while trying to figure out why the project was compiling with compose beta
The extension version at least seems mandatory